content/posts/2023-12-04-matura-bel-top-schools.md

```diff
@@ -0,0 +1,54 @@
aJZbrPvNT4v180yl00AbtPM+wbdiHDQzSUJZfRFleSLuiJnB7+I+n8SqZeKJ8+mp3WXOyQXB9erBYz+QOnjpHDoLdeJlouioOaH+FTygzeS3V5nJbvZlClk6mYm3f/gBz8rJwUAAGCO5nGeZSqnF6LlzHnWQIUvthufd5kryM9KmddwJrpKrRXF1ahS93Zmk3jMSicKMgTJRbzTJpBwumywYNJjalLmGHsp2l7OsWi2dzG5X4tyNlgr8TGmFx0zWzXv5pzzx3kwotYdA8GHSPJcSyTKD7EqwAE3n1EwDsfdGaIv+FfLJ2VJEsloyXA9d7lD3cBC9wse/dAekrPhA3lGAwZlI48mKut311G0n15cSN2jBQQAAJi3GZ9nRbRJfWiWy7vAc7R77lXY9VV994WoqbsGH6pa2c2q1GG2jv3Psh16crb6VnQu3Sy73J6OqckFyX1m1+gkuNCSgYCAV5Hn9L7jyRuZy8X1h6Y+vlfpteToBjwke7loWieZ5iUJ8IEDbC4BCP3ia+T08beHujR02Yh8R+Ru6umokDBf8NwmeFHhFkcnq+nopHm+OXaTp3t9zRVH/mCayClRTQAAMFfzOM/SpvXmdipvOXmQnCN5xQER7UuvuSLMGWo7XJFTLkdIwHgmTLg+8TE1iejiYZwYs12Um83szzvp/Us/PqpdQvKXezd6kB65pfc4PxdcJZlWdsQXYB+bTwuIEdJNB7W/WGpSNXmuVbgCLw4c3FuRY7IHUcAiTIIQOU2momF6kkireTMfFwAAgAU2j/OsJPmfNot/J2+eJAFgPfn3emVXNrQ/f3KuheU003N3edg9VoovOnb6W+SkLzQW5hpJAmgD5/x2yM38bSavv2wXJGA/W7gARGrYnLgPVlbkOxzZkqGdzDe61cIIqX5pW4M/gKkM0wAAAAtvHudZ2qpB9OqyVt5W8rtTdDqpil8oaxokMfu4FY3W8QJN15faDM/dSxkMFhwdb7vRKCA90eOju3JHAIy0UAGI1I9i9COlYwLLvD2Mfii12ZQGG260z8g1zZ6r9/q/xP0T57+fAAAABeZxnpUanlATgZ8tykmhuR7CB3Ks+rZ4ejMBiFPmpsMW6pDoW6W6wwKzNjq4AiBlYQIQqdYEgeiQUZX+IaPSKt7I/mGjm0KVpPtgfgRPRcN2xs2uVpN91OaLV7R/ot3OQD9KAACAhTCP86xUl4p2NKxhQZ98K71POr+51SRO/FdNhgzFEprZufvDkssPtLRIB8YKA13Jcl46l8k43UdGdUUBDoBDsgCSsYU1Iu9Hke4Hsm5+cIYLkOTL3vFGfsH96O+D6X3Jo+E2Je6XqAVT33jaz0cJbyhUAADAwpn1eZZtaWGCD/GwjRrs+Ork50hmPW+YCl8wOGQolsDsz91bdn2D5+t9u1VJjqVOanvaksGT6tFKlN8hO5ARxuf8nbAvT0Q8b5xgMi/IthotXuFiJTD3FhCpoZb8aLxeE+nOHa3iQfylNvM/l7e+1HA/rbFHvchb50Duh8H1mh/F55JtNgUAAGBBzPo8K2lpoaMIBDoUaHhfTuUFH7SCqOszt3XB/jTjc/dk+Vb0z6GC0enCZFqlryVDtK/pITYHdQMXlV6gIewkiVXD7vCfQ7rBs87oVkDAfjfXAER3qCUvim6fTloZ5DKFynU7TE+3sBpgvuDn9N4UdE2ZAs33oLke8rbXHQVDC823KVQAAMBimPV5li7jHYqCD1GwQ4cCLaxQHpWquYizY26Xi0a6MLsfVV7Na3lXsFTmce5uAgJvpNc7SEdWiYbq1HP3VBJWO0KLOR5fylwuNfJdOgBh1rMd38m5rO4b0agzYTeJalOAA24uAQj9ct79hGwl/QKvjzPOc9GoFBrlt8NKdVbkkkxBqjAqHAUjyhEBAAAwZ/M4z7p7Ri5oMCEKdnhydlSwQyUtI5rR9nSki4wgRDSChhdV3trmiva2YOnM+tzdBgR0vXc+0d+6Jjp3l2RI2E6U4LRHE8/HwZIgGbklvVzVnvNHo8ekWvVE3624q0m1ciRjuUNyOSvgARxUnrnCH+oDzTYsMxJFyL1k+Er9oleKx801+3a6u2yvAIibTplCxkRMNbmRL0lUMmpiOPDDp837vIpc6Ft3JyoMqkP7kNpm1vZMgfWuJtOJoplmee1PmSRJAgAAmKt5nGeZ88kbovOM2p7xoYcSPPJHcR/6O580FcKO2Z4d1lC3p03ow2iUseg8K9rmrpw/9kfSEMyNtgpO8hz09D63Vt/THbmio5ro40mPqb0w34G6+Q7YYEDTS7plpI6p65pkPmO5dW2Rk/zb8pIAWXe5nJwmSZ6Va7nLaSLWlfzuSMBBYOMO809CGQcAqmVn16Z8piA7LYdlU5s6JeNEW+2kANvMWLQq4UCh6Y3eh+72VkxBVom3ZwqmeJted3t1AQAAWDSzO8+abHumQmaCEKdl15xnibzUt704wXfT/H/JBB+agnnzh86lreHnu8fAno+pCei5uQkKtKIgRChB6piKthe1dshebtMs106W823rjO6xuGICYRlBBO2GbY7jZ7yHsmXmDYaWO5S9HHAQzaUFxDRpxDH6oXsYJa5piWOz3h4AAMC8zO08S4c2fDC9hOJYHMtyTGnrHLOP/tjLBdGQsascw0A/G3dY+gAEAAAAAABYXDbuMPdhOAEAAAAAwP5HAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADhHAAIAAAAAADjntc9IKAAAAAAAAA7RAgIAAAAAADj3fwENkuRRBhtB3QAAAABJRU5ErkJggg==)
```

Comment:
Instead of inline binary, please add graphics as separate files in [/static/img/posts/](https://github.com/data-for-good-bg/data-for-good-bg.github.io/tree/blog-post-bel-2023/static/img/posts), add a folder `2023-12-04-matura-bel-top-schools/` and put the files there. Then refer them in the markdown.

Reply:
Images here: static/img/posts/2023-12-04-matura-bel-top-schools, referenced in markdown